streamer/src/nonblocking/quic.rs

```diff
@@ -331,6 +331,7 @@ async fn run_server(
                 stats
                     .connection_rate_limited_across_all
                     .fetch_add(1, Ordering::Relaxed);
+                connection.ignore();
```

Comment:
use connection.ignore which can more efficiently drop connection without notifying the clients.